### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by cypherhoodlum** [NostrGit/NostrGit#82] [General]
Many design decisions will be made on the fly during the lifespan of this project. Many people will have different ideas on how to do things. I propose we create markdown documents only into the documentation folder in the repo root. When you notice you have made a design decision, create a .md file for it in the documentation folder and try to keep it organized. Always search for duplicates before creating a new one. How's that sound?

**Comment by PeerRich** [comment]
i think thats a good solution.

speaking of design, i find ui.shadcn.com the best way to move quickly with good looking UI

**Comment by pierresisson** [comment]
a few example to start

- pull requests naming convention (ex: always start with a described list : `Add`, `Fix`...)
- branches naming convention (ex: always start with `username:add-mobile-layout`)
- components architecture
- github action (prettier, stylelint...)

**Comment by cypherhoodlum** [comment]
I'm using a lot of stuff from the bitcoindevkit/bdk contribution guide because it's awesome. I'll append anything you come up with in this Issue.
